pkg/tsdb/influxdb/flux/executor_test.go

```diff
@@ -273,3 +285,12 @@ func TestBuckets(t *testing.T) {
 		fmt.Println("----------------------")
 	})
 }
+
+func TestGoldenFiles(t *testing.T) {
+	t.Run("Renamed", func(t *testing.T) {
+		_, err := verifyGoldenResponse("renamed")
+		if err != nil {
+			t.Fatal(err.Error())
+		}
```

Comment:
require.NoError(t, err)

Reply:
no objection; only that this is already merged in master in #26329 if we accept the suggestion here we need to change it also in the master.

Reply:
Good point @papagian. Might as well not change this.